undertow/src/main/java/org/wildfly/extension/undertow/logging/UndertowLogger.java

```diff
@@ -312,4 +313,8 @@
 
     @Message(id = 72, value = "Could not find configured external path %s")
     DeploymentUnitProcessingException couldNotFindExternalPath(File path);
+
+    @LogMessage(level = DEBUG)
```

Comment:
Debug level messages do not need to be internationalised, so there is no need to put them on the logger.